src/app.css

```diff
@@ -26,3 +26,8 @@ body::before {
 	--shadow-gray: 0 0 10px 10px rgba(0, 0, 0, 0.08);
 	--color-oba-green: oklch(0.632 0.1728 138.8);
 }
+
+.right-alignment {
```

Comment:
instead of applying `right-alignment` throughout the codebase, let's use the dir attribute on the html tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/dir